code/modules/clothing/suits/labcoat.dm

```diff
@@ -1,6 +1,8 @@
 /obj/item/clothing/suit/storage/toggle/labcoat
 	name = "labcoat"
 	desc = "A suit that protects against minor chemical spills."
+	icon = 'icons/mob/clothing/suit/labcoat.dmi'
```

Comment:
Have you tested Bulwarks still being able to wear labcoats? They have unique sprites, make sure to add a reference for their sprite sheet.

Reply:
addressed

also labcoats are a child of regular ol' suits so i don't have to do much.